unlock-app/src/components/interface/locks/metadata/index.tsx

```diff
@@ -229,6 +229,7 @@ export function UpdateMetadataForm({ lockAddress, network, keyId }: Props) {
   const isLockSelected = selected.lockAddress && selected.network
 
   const isTokenURIEditable = useMemo(() => {
+    return true
```

Comment:
is this necessary?

Reply:
oups! No it is just to make debugging easier!